packages/slidev/node/virtual/shiki.ts

```diff
@@ -70,6 +70,20 @@ export const templateShiki: VirtualModuleTemplate = {
       `  langs: [${langsInit.join(',')}],`,
       `  loadWasm: import('${await resolveImportUrl('shiki/wasm')}'),`,
       '})',
+
+      'let highlight',
+      'export async function getHighlighter() {',
+      '  if (highlight) return highlight',
+      '  const highlighter = await shiki',
+      '  highlight = (code, lang, options) => highlighter.codeToHtml(code, {',
+      '    lang,',
+      `    theme: ${JSON.stringify(themeOptionsNames.theme)},`,
+      `    themes: ${JSON.stringify(themeOptionsNames.themes)},`,
+      '    defaultColor: false,',
+      '    ...options,',
```

Comment:
Moved here from https://github.com/slidevjs/slidev/blob/4707949b6612a0f1cc60f8c969ea7dadf4727862/packages/client/setup/code-runners.ts#L20-L28.

It now uses Shiki's dual theme feature. I am not sure why it wasn't used.